### DISCUSSION TITLE
We don't support the chainId you provided: eip155:80001. See the list of supported chains here: https://docs.walletconnect.com/cloud/blockchain-api#supported-chains

### DISCUSSION THREAD

**Discussion by zhupeikang** [WalletConnect/walletconnect-monorepo#4473] [Developer Support] [answered]
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/24474217/bf0c01f3-c4e4-46f4-9a7c-c709b15f0d44)

**Comment by zhupeikang** [comment]
请问现在不支持这个链了吗？

**Reply by haiifeng** [reply]
same issus
<img width="1433" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/20502943/bee3836b-288e-4dd9-bb43-318a1ebe49cb">

**Reply by geekbrother** [reply, marked as answer]
eip155:200901 isn't supported. Please check [the list](https://github.com/WalletConnect/blockchain-api/blob/master/SUPPORTED_CHAINS.md).

**Comment by geekbrother** [comment]
Mumbai (80001) is not supported anymore. 
Please check https://github.com/WalletConnect/blockchain-api/blob/master/SUPPORTED_CHAINS.md
